chaos_genius/databases/models/kpi_model.py

```diff
@@ -6,6 +6,10 @@
 
 from chaos_genius.databases.base_model import Column, PkModel, db
 
+from chaos_genius.databases.models.data_source_model import DataSource
+
+UNSUPPORTED_DATASOURCES_FOR_MULTIDIM = ["Druid"]
```

Comment:
Why are we adding this in the KPI model? Let's not define this here. If you are using this then define this in the `chaos_genius/utils/metadata_api_config.py`

Reply:
import Datasource is okay ? just the variable need to be move ?